src/src/WebServer/Rules.cpp

```diff
@@ -552,6 +552,7 @@ void Rule_showRuleTextArea(const String& fileName) {
   addHtml(F("<textarea id='rules' name='rules' rows='30' wrap='off'>"));
   size = streamFromFS(fileName, true);
   addHtml(F("</textarea>"));
+  addHtml(F("<script>initCM();</script>"));
```

Comment:
There is a function to add scripts and also to serve it directly where it checks to see if the static file is included during build and if not, it will serve it via the CDN links
I will try to add it for you, pushing on top of this PR.

Reply:
Great...